README.md

```diff
@@ -221,22 +221,43 @@ Gathers the matching credentials that fit a given presentation definition
 
 #### Return value
 - If the selection was successful or partially successful, the `matches` array will consist of `SubmissionRequirementMatch` object(s), representing the matching credentials for each `SubmissionRequirement` in the `presentationDefinition` input parameter.
-- If the selection was not successful, the `errors` array will consist of `Checked` object(s), representing what has failed in your selection process. 
+- If the selection was not successful, the `errors` array will consist of `Checked` object(s), representing what has failed in your selection process.
 
 ```typescript
+import { Status } from './ConstraintUtils';
+
 interface SelectResults {
-  errors?: Checked[]
+  errors?: Checked[];
   matches?: SubmissionRequirementMatch[];
+  /**
+   * This is the parameter that pejs library user should look into to determine what to do next
+   * Status can have three values:
+   *  1. INFO: everything is fine, you can call `submissionFrom` after this method
+   *  2. WARN: you probably have sent us more than you should, this will be fine in our side,
+   *  but not good for your end user, therefore you should not send some credentials
+   *  3. Error: the credentials you've sent didn't satisfy the presentationDefinition object
```

Comment:
We can perhaps change the text to 
Error: the credentials you've sent didn't satisfy the requirement defined presentationDefinition object